lib/posix-process/vfork.c

```diff
@@ -18,21 +18,17 @@
 
 UK_LLSYSCALL_R_E_DEFINE(pid_t, vfork)
 {
-	struct posix_process *child_proc;
 	struct clone_args cl_args = {0};
-	pid_t child_tid;
+	pid_t pid; /* child */
```

Comment:
Commit subject is a bit misleading, maybe rewrite as to imply it refers to fixing nomenclature rather than C-type

Reply:
Good point, will update 👍🏼